src/utils/index.ts

```diff
@@ -203,6 +203,10 @@ export function isPairOnList(pairs: Pair[], pair?: Pair): boolean {
   return !!pairs.find(loopedPair => loopedPair.equals(pair))
 }
 
+export function firstLetterCapital(value: string): string {
+  return value.charAt(0).toLocaleUpperCase() + value.slice(1)
+}
+
```

Comment:
We can use CSS for this. Not need for extra code and logic :)

Reply:
I think this is a better solution since I would just need to pass one more extra parameter to the footer and also one more extra styling component just for that single piece of text

Reply:
You can add a prop, or use composition to create a new component based on the base style. Just saying that we don't need ifs and loops for capitalize text.

Reply:
Not sure what you are suggesting we do here if you can write some piece of code would be helpful keep in mind we are talking about one word here. I didn't use any ifs or loops xD

Reply:
Ahahaha, not hating. Just saying we have unecessary code. .splice() uses a for loop inside to create a new string, for example. CSS is lighter and faster, and with styled components is easier to implement. Just don't want bad code that other might use with no need. This answer on [Reddit](https://www.reddit.com/r/javascript/comments/9w6qbz/tricky_question_what_is_faster_css_texttransform/e9kmsx6/?context=3) sums it up. I always prefer CSS over JS when styling the page/components.

Reply:
I'm leaning toward the CSS solution, this function seems to handle `LimitOrder` enum, so why not create a map? A hashtable that has time complexity is O(1). Pick your fighter @Mi-Lan

Reply:
Imma just think outside of box
![](https://media.giphy.com/media/s0FsE5TsEF8g8/giphy.gif)

Reply:
https://github.com/levelkdev/swapr-dapp/pull/1633/commits/5b79ea366531407ffb384051d5c4dd9615204428
@adamazad @berteotti